application/src/utils/formatTotalAmount.test.ts

```diff
@@ -0,0 +1,21 @@
+import { formatTotalAmount } from './formatTotalAmount';
+
+describe('formatTotalAmount function', () => {
+  let value;
+  beforeEach(() => {
+    value = 0;
+  });
+  test('format 1.5555555 to 1.56', () => {
```

Comment:
we shouldn't put expected values to the name of test.

Reply:
fixed